EF.Core.Training.CodeShop/Services/ApiContext.cs

```diff
@@ -10,7 +10,7 @@ public class ApiContext : DbContext
         public ApiContext()
         {
             // saves the SQLite EF.Core.Training.db to the solution's root folder
-            string path = Path.GetFullPath(Path.Combine(AppContext.BaseDirectory, @"..\..\..\..\"));
+            string path = Path.GetFullPath(Path.Combine(AppContext.BaseDirectory, @"../../../../"));
```

Comment:
I completed this challenge on a Mac.
Let me tell you...my `dotnet ef` CLI did __not__ like those back slashes.